__tests__/testUtilities/stateUtils.js

```diff
@@ -247,6 +247,24 @@ const buildResourceWithLiteral = (state, options) => {
     ]
   }
 
+  if (options.hasIntegerValidation) {
+    state.entities.propertyTemplates[
+      "ld4p:RT:bf2:Title:AbbrTitle > http://id.loc.gov/ontologies/bibframe/mainTitle"
```

Comment:
inappropriate propertyTemplate for these sorts of literals, but gud enuf for testing ?